.github/workflows/ios-cloud-build.yml

```
@@ -24,7 +24,7 @@ jobs:
       with:
         ssh-private-key: ${{ secrets.SSH_PRIVATE_KEY }}
     - name: Setup Ruby
-      uses: actions/setup-ruby@v1
+      uses: ruby/setup-ruby@v1
       with:
         ruby-version: 2.6.x
```

- I would upgrade Ruby version to a more up-to-date one.
- We can use the action for caching instead of the separate cache step.
- These lines shouldn't be required in the case of the automatic cache:

```
gem install bundler
bundle config path vendor/bundle
```

```suggestion
        ruby-version: 2.7
        bundler-cache: true
```